### PR TITLE
Review bundle

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by ngilain** [comment]
:+1:

File: WorkflowFunction/Manager/AuthorizationWorkflowRightManager.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace OpenOrchestra\WorkflowFunction\Manager;
 
+use Doctrine\Common\Collections\ArrayCollection;
```
Comment (nicolasThal) [inline WorkflowFunction/Manager/AuthorizationWorkflowRightManager.php:5]: you should use `Doctrine\Common\Collections\Collection` instead, it allows to have more collection type (PersistentCollection for instance)

File: WorkflowFunction/Manager/AuthorizationWorkflowRightManager.php [hunk 2]
```diff
@@ -29,35 +31,42 @@ public function __construct($authorizationClass)
...
-            $authorization->setReferenceId($indexReferences[$add]->getId());
+            $authorization->setReferenceId($add->getId());
             $workflowRight->addAuthorization($authorization);
         }
 
         return $workflowRight;
     }
+
+    /**
+     * @param array|ArrayCollection  $list
```
Comment (nicolasThal) [inline WorkflowFunction/Manager/AuthorizationWorkflowRightManager.php:55]: You can then change to `array|Collection`

File: WorkflowFunction/Model/AuthorizationInterface.php [hunk 3]
```diff
@@ -20,7 +20,7 @@ public function setReferenceId($referenceId);
     public function getReferenceId();
 
     /**
-     * @param ArrayCollection
+     * @param ArrayCollection $workflowFunctions
```
Comment (nicolasThal) [inline WorkflowFunction/Model/AuthorizationInterface.php:23]: smae

File: WorkflowFunctionAdminBundle/Facade/WorkflowFunctionCollectionFacade.php [hunk 4]
```diff
@@ -34,7 +34,7 @@ class WorkflowFunctionCollectionFacade extends AbstractFacade
     public $recordsFiltered;
 
     /**
-     * @param FacadeInterface $log
+     * @param FacadeInterface $workflowFunction
```
Comment (nicolasThal) [inline WorkflowFunctionAdminBundle/Facade/WorkflowFunctionCollectionFacade.php:37]: same

File: WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php [hunk 5]
```diff
@@ -41,7 +41,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
-     * @param OptionsResolver $resolver
+     * {@inheritdoc}
```
Comment (nicolasThal) [inline WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php:44]: put the real doc instead

File: WorkflowFunctionAdminBundle/Form/Type/WorkflowRightType.php [hunk 6]
```diff
@@ -38,7 +38,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
-     * @param OptionsResolverInterface $resolver
+     * {@inheritdoc}
```
Comment (nicolasThal) [inline WorkflowFunctionAdminBundle/Form/Type/WorkflowRightType.php:41]: real doc

File: WorkflowFunctionBundle/Document/WorkflowRight.php [hunk 7]
```diff
@@ -91,7 +91,7 @@ public function getUserId()
     /**
      * Set authorizations
      *
-     * @param ArrayCollection
+     * @param ArrayCollection $authorizations
```
Comment (nicolasThal) [inline WorkflowFunctionBundle/Document/WorkflowRight.php:94]: same `Collection usage`

**Pull request merged**